### DISCUSSION TITLE
qt_viewer example cannot load slint file

### DISCUSSION THREAD

**Discussion by 996666925** [slint-ui/slint#2702] [Q&A]
qt_viewer example cannot load slint file, and the error message is "QObject::setParent: Cannot set parent, new parent is in a different thread".how to solve the problem？

**Comment by ogoffart** [comment]
I cannot reproduce this.  What platform are you running on? What's your version of Qt?
Are you able to debug and run the program in a debugger with `QT_FATAL_WARNINGS=1` to get a backtrace when this happens?

The qt_viewer doesn't use threads so i'm really wondering why you would get this error.
